### DISCUSSION TITLE
request to submit to Flathub as an extension

### DISCUSSION THREAD

**Discussion by mdmundo** [1player/host-spawn#25] [General]
Hey, I am writing to inform you that I am submitting your software to [Flathub as a vscode extension](https://github.com/flathub/flathub/pull/4267) as I must confirm that:

> I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.

**Comment by 1player** [comment]
Sure, looks good to me. I'll comment on the PR
